### PR TITLE
add meta tags and links to fonts/styleesheet/css

### PR DESCRIPTION
**Pull request by Anshumaan-Tripathi**
changes: - 
added meta tags
added image link to Og tags 
added styles link and fonts link

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -245,13 +270,13 @@ <h2 class="experience">Submit Your Experience</h2>
           </div>
           <div>
             <label for="agreement">
-              <input type="checkbox" id="agreement"/>
+              <input type="checkbox" id="agreement" />
```
Comment (anshumaantripathi) [inline src/index.html:273]: remove this unnecessary  change

Comment (anshumaantripathi) [inline src/index.html:273]: this is also unnecessary change

File: src/index.html [hunk 2]
```diff
@@ -2,28 +2,53 @@
 <html lang="en">
   <head>
     <title>Treetop Tea Experience</title>
```
Comment (anshumaantripathi) [inline src/index.html:4]: add nature word in title may be more creative

**Review by anshumaantripathi — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (Anshumaan-Tripathi) [reply src/index.html]: i think this part is now fine

File: src/index.html (hunk 1 above)
Comment (Anshumaan-Tripathi) [reply src/index.html]: this is done

Comment (Anshumaan-Tripathi) [reply src/index.html]: done

**Review by anshumaantripathi — APPROVED** [review]
